systemd/yamuisplash.service

```diff
@@ -1,14 +1,16 @@
 [Unit]
 Description=Simple yamui splash screen for SailfishOS
 Before=droid-hal-init.service
+Before=sailfish-unlock-agent.service
+Before=start-user-session.service
 After=tmp.mount
 DefaultDependencies=no
 # To avoid starting again when swichting from emergency-call.target to graphical.target
 ConditionPathExists=!/tmp/.yamuisplash.done
 
 [Service]
-Type=simple
+Type=notify
+Restart=no
 EnvironmentFile=-/var/lib/environment/compositor/*.conf
-ExecStart=/bin/sh /usr/bin/yamuisplash
-# make sure we run for at least 5 seconds for any touch initialization sequences to finish.
-ExecStartPost=/bin/sleep 5
+ExecStartPre=/bin/touch /tmp/.yamuisplash.done
```

Comment:
Not sure if /tmp is the best place for this, something like /run/yamui/done would be better.

Reply:
Possibly. Did not think about it, just tweaked where it does what it already did. But this is executed quite early in bootup -> is /run already ready for use? And if it is, where/how should this /run/yamui directory be created?

Come to think about it: with Restart=no, is it needed at all? (ttbomk the binary exits, service stays started but dead -> requires manual restart before anything happens)

Reply:
Apparently the flag file might be relevant in some exotic corner case -> still use it, but moved it under /run